recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz  # [linux64]
```

Comment:
This new tarball does not include binaries anymore (hurray!)

However, there is quite a bit of third-party code vendored in the tarball, such as the entire vscode codebase, together with vscode typings and vscode.proposed.

I think we should probably only include the required sources in the tarball.

Reply:
We are pulling in code from Visual Studio Code - Open Source (https://github.com/microsoft/vscode), making some changes on top and building the entire codebase. So, we are including the entire source and have not specifically removed packages/components that might not be needed.

Reply:
The github repo would definitely help with the review process!

Reply:
Yes, we're actively working on setting up the github repo. There are some company processes to navigate, so it's taking a bit of time. So as to not block this PR, if you need any info on the repo setup please let me know.

Meanwhile, I'll be addressing all other comments on this PR.